lib/watcher.js

```diff
@@ -58,9 +59,13 @@ function Watcher(logger, api, files, sources) {
 		}
 
 		var self = this;
-		this.busy = api.run(specificFiles || files, {
+		var options = {
 			runOnlyExclusive: runOnlyExclusive
-		}).then(function (runStatus) {
+		};
+		if (extensions) {
```

Comment:
This is an internal API. We can just make passing the `extensions` argument required and drop this if-statement.